tests/sdk/queries/alerts/filters/test_alert_filter.py

```diff
@@ -35,7 +34,7 @@ def format_timestamp_with_microseconds(test_time):
 
 def format_datetime_with_microseconds(test_date):
     prefix = test_date.strftime(MICROSECOND_FORMAT)
-    timestamp_str = "{}".format(prefix)
+    timestamp_str = f"{prefix}"
```

Comment:
I don't think `prefix` needbe interpolated, it is already a string

Reply:
Unless this is supposed to be `f"{prefix}Z" like in `test_file_event_filter.py`.
I am not sure!